chrome/browser/net/chrome_network_delegate.cc

```diff
@@ -327,6 +330,98 @@ int ChromeNetworkDelegate::OnBeforeURLRequest(
   return rv;
 }
 
+std::string generateBlockedElemhideJavaScript(std::string url, std::string relative_url) {
+  std::stringstream ss;
```

Comment:
you could just return the string literal here.

Reply:
later `std::string` (`base::string16`) is required to call `ExecuteJavaScript..`. Is there any point to return literal and then convert to `base::string16`?

Reply:
My point it that usimng `std::stringstream` here is not optimal, whether you return a `std::string` or the result of `base::UTF8ToUTF16()`.